server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerCountersResource.java

```diff
@@ -15,6 +20,30 @@
 public class CacheContainerCountersResource extends SimpleResourceDefinition {
     static final PathElement PATH = PathElement.pathElement(ModelKeys.COUNTERS, ModelKeys.COUNTERS_NAME);
 
+    //TODO Could not find these definitions anywhere in counter API
+    private static final String AVAILABLE = "AVAILABLE";
+    private static final String CONSISTENT = "CONSISTENT";
```

Comment:
These constants are defined in the following enum `org.infinispan.counter.configuration.Reliability`